### PR TITLE
 [Bug 20538] Ensure flushEvents("all") works on Mac

### PR DESCRIPTION
**Pull request by livecodepanos**

### CODE REVIEW

File: engine/src/desktop-dc.cpp [hunk 1]
```diff
@@ -911,9 +911,10 @@ void MCScreenDC::flushevents(uint2 e)
 		t_mask = kMCPlatformEventKeyDown;
 	else if (e == FE_KEYUP)
 		t_mask = kMCPlatformEventKeyUp;
-	
-	if (t_mask != 0)
-		MCPlatformFlushEvents(t_mask);
+    else if (e == FE_ALL)
+        t_mask = kMCPlatformEventMouseDown + kMCPlatformEventMouseUp + kMCPlatformEventKeyDown + kMCPlatformEventKeyUp;
```
Comment (runrevmark) [inline engine/src/desktop-dc.cpp]: The kMCPlatformEvent constants are flags, so you should use bitwise-or here (|) rather than +.

**Comment by runrevmark** [comment]
@livecode-vulcan review ok 7eacbdf

**Comment by livecode-vulcan** [comment]
💙 review by @runrevmark ok 7eacbdfee59553a4bf2e3ecb02e95b78c778ff34

**Comment by livecode-vulcan** [comment]
😎 test success 7eacbdfee59553a4bf2e3ecb02e95b78c778ff34

* try-community-armv6-android-api8: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-api8/builds/925)
* try-community-armv6-android-api9: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-api9/builds/941)
* try-community-js-emscripten-sdk1.35: [success](https://vulcan.livecode.com/builders/try-community-js-emscripten-sdk1.35/builds/915)
* try-community-universal-ios-iphoneos11.0: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphoneos11.0/builds/90)
* try-community-universal-ios-iphonesimulator11.0: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphonesimulator11.0/builds/91)
* try-community-universal-mac-macosx10.6: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.6/builds/919)
* try-community-universal-mac-macosx10.9: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.9/builds/941)
* try-community-x86-linux-debian7: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian7/builds/925)
* try-community-x86-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian8/builds/941)
* try-community-x86_64-linux-debian7: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian7/builds/924)
* try-community-x86_64-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian8/builds/934)
* try-community-x86-win32: [success](https://vulcan.livecode.com/builders/try-community-x86-win32/builds/956)
* try-community-x86_64-win32: [success](https://vulcan.livecode.com/builders/try-community-x86_64-win32/builds/920)

**Pull request merged**